docs/clay_over_aoi.ipynb

```diff
@@ -0,0 +1,727 @@
+      "  times = pd.to_datetime(\n",
+      "Subsetting to [1500, 1500, 2524, 2524]\n",
+      "Writing tempfiles to  data/chips\n",
+      "No bucket specified, skipping S3 sync.\n",
+      "Too much cloud coverage\n",
+      "No bucket specified, skipping S3 sync.\n",
+      "2 name                                                    45QUC\n",
+      "Region                                                   Puri\n",
+      "geometry    POLYGON ((85.0907496721 19.7978447286, 85.0832...\n",
+      "Name: 2, dtype: object\n",
+      "Starting algorithm for MGRS tile 45QUC with index 2\n",
+      "Processing data for date range 2020-01-01/2020-04-01\n",
+      "Found 16 Sentinel-2 items\n",
+      "EPSG code based on Sentinel-2 item:  32645\n",
+      "Searching S1 in date range 2020-02-09/2020-02-15\n",
+      "Found 2 Sentinel-1 items\n",
+      "Most overlapped orbit:  descending\n",
+      "Found 4 DEM items\n",
+      "/home/tam/apps/miniforge3/envs/claymodel/lib/python3.11/site-packages/stackstac/prepare.py:408: UserWarning: The argument 'infer_datetime_format' is deprecated and will be removed in a future version. A strict version of it is now the default, see https://pandas.pydata.org/pdeps/0004-consistent-to-datetime-parsing.html. You can safely remove this argument.\n",
+      "  times = pd.to_datetime(\n",
+      "/home/tam/apps/miniforge3/envs/claymodel/lib/python3.11/site-packages/stackstac/prepare.py:408: UserWarning: The argument 'infer_datetime_format' is deprecated and will be removed in a future version. A strict version of it is now the default, see https://pandas.pydata.org/pdeps/0004-consistent-to-datetime-parsing.html. You can safely remove this argument.\n",
+      "  times = pd.to_datetime(\n",
+      "/home/tam/apps/miniforge3/envs/claymodel/lib/python3.11/site-packages/stackstac/prepare.py:408: UserWarning: The argument 'infer_datetime_format' is deprecated and will be removed in a future version. A strict version of it is now the default, see https://pandas.pydata.org/pdeps/0004-consistent-to-datetime-parsing.html. You can safely remove this argument.\n",
+      "  times = pd.to_datetime(\n",
+      "Subsetting to [1500, 1500, 2524, 2524]\n",
+      "^C\n",
+      "\n",
+      "Aborted!\n",
+      "3 name                                                    45QVC\n",
+      "Region                                                   Puri\n",
+      "geometry    POLYGON ((86.04044162970094 20.52127966663557,...\n",
+      "Name: 3, dtype: object\n"
+     ]
+    },
+    {
+     "name": "stderr",
+     "output_type": "stream",
+     "text": [
+      "\n",
+      "KeyboardInterrupt\n",
+      "\n"
+     ]
+    }
+   ],
+   "source": [
+    "for index, row in mgrs_aoi.iterrows():\n",
+    "    print(index, row)\n",
+    "    ! python scripts/datacube.py --sample data/mgrs/mgrs_aoi.fgb --subset 1500,1500,2524,2524 --localpath data/chips  --index {index} --dateranges 2020-01-01/2020-04-01,2021-06-01/2021-09-15"
```

Comment:
This line requires a `PC_SDK_SUBSCRIPTION_KEY` token to be set I think, otherwise users will get a `requests.exceptions.HTTPError: 403 Client Error: Forbidden for url: https://planetarycomputer.microsoft.com/api/sas/v1/token/sentinel1euwestrtc/sentinel1-grd-rtc`. Need to document this inline perhaps, xref #119.

Reply:
Added reference to the markdown